src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -735,22 +738,24 @@ static void _toColor(const char* str, uint8_t* r, uint8_t* g, uint8_t* b, char**
                     tb /= 100.0f;
                     brightness = _skipSpace(brightness + 1, nullptr);
                     if (brightness && brightness[0] == ')' && brightness[1] == '\0') {
-                       if (_hslToRgb(th, ts, tb, r, g, b)) return;
+                       return _hslToRgb(th, ts, tb, r, g, b);
                     }
                 }
             }
         }
+        return true;
```

Comment:
please remove this.
If hsl color parsing is successful, it should return at line 741. If line 741 is not touched, _toColor() should be fail.